includes/backends/class-omise-backend-installment.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Note: all of calculations in this class are based only on Thailand VAT and fee
+ *       as currently Installment feature supported only for merchants
+ *       that have registered with Omise Thailand account.
+ *
+ * @since 3.4
+ *
+ * @method public initiate
+ * @method public get_available_providers
+ * @method public filter_available_terms
+ * @method public calculate_monthly_payment_amount
+ */
+class Omise_Backend_Installment extends Omise_Backend {
+	/**
+	 * @var array  of known installment providers.
+	 */
+	protected static $providers = array();
+
+	public function initiate() {
+		self::$providers = array(
```

Comment:
I intended to have this `initiate` here as normally WordPress project's behaviour is to have a place where 3rd-party developers can hook and alter its value (something like `apply_filter()`). – _just leave a room for its possibility but don't have any real request or a use-case at the moment._